Content.Shared/Disposal/Components/SharedDisposalUnitComponent.cs

```diff
@@ -33,41 +33,44 @@ public abstract partial class SharedDisposalUnitComponent : Component
     /// <summary>
     /// State for this disposals unit.
     /// </summary>
-    [DataField("state")]
+    [DataField]
     public DisposalsPressureState State;
 
     // TODO: Just make this use vaulting.
     /// <summary>
     /// We'll track whatever just left disposals so we know what collision we need to ignore until they stop intersecting our BB.
     /// </summary>
-    [ViewVariables, DataField("recentlyEjected")]
+    [ViewVariables, DataField]
     public List<EntityUid> RecentlyEjected = new();
 
     /// <summary>
     /// Next time the disposal unit will be pressurized.
     /// </summary>
-    [DataField("nextPressurized", customTypeSerializer:typeof(TimeOffsetSerializer))]
+    [DataField(customTypeSerializer:typeof(TimeOffsetSerializer))]
     public TimeSpan NextPressurized = TimeSpan.Zero;
 
     /// <summary>
     /// How long it takes to flush a disposals unit manually.
     /// </summary>
-    [DataField("flushTime")]
+    [DataField]
     public TimeSpan ManualFlushTime = TimeSpan.FromSeconds(2);
```

Comment:
Be careful with datafield strings because this updates the field from FlushTime to ManualFlushTime.

Reply:
oops yea, didnt notice that. maybe i should just go through the prototypes and remove these datafields for shareddisposalunit but make sure its all changed correctly in yaml.